angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts

```diff
@@ -15,5 +15,10 @@ export class NimbleLabelProviderCoreWithDefaultsDirective {
         this.elementRef.nativeElement.popupDismiss = $localize`:Nimble popup - dismiss|:Close`;
         this.elementRef.nativeElement.numericDecrement = $localize`:Nimble numeric - decrement|:Decrement`;
         this.elementRef.nativeElement.numericIncrement = $localize`:Nimble numeric - increment|:Increment`;
+        this.elementRef.nativeElement.errorIcon = $localize`:Nimble icon - error|:Error`;
```

Comment:
The Blazor label providers are missing all these too. Wanna handle it in this PR (seems reasonable) or separate?

Reply:
Added Blazor and updated instructions in #1747 to include it too.